src/main/java/racingcar/logic/RacingCarController.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.logic;
+
+import java.util.HashMap;
+import java.util.List;
+import racingcar.io.Input;
+import racingcar.io.Output;
+import racingcar.message.Message;
+
+public class RacingCarController {
+
+    Input input = new Input();
+    Output output = new Output();
+    RacingCarService racingCarService = new RacingCarService();
+
+    String carList;
+    Integer attemptCount;
+    HashMap<String, Integer> result;
```

Comment:
```suggestion
    Map<String, Integer> result;
```
확장성을 위해 인터페이스로 선언하고 초기화를 구현체로 해주는 것이 좋아보여요!

Reply:
오 저는 습관적으로 List,Map으로 썼는데 이런 이유가 있었네요... 배우고 갑니당

Reply:
오오 새로운 지식 배워갑니다!!